keeper-gradle-plugin/build.gradle.kts

```diff
@@ -78,11 +80,20 @@ val agpVersion = findProperty("keeperTest.agpVersion")?.toString() ?: defaultAgp
 
 // See https://github.com/slackhq/keeper/pull/11#issuecomment-579544375 for context
 val releaseMode = hasProperty("keeper.releaseMode")
+val shade: Configuration = configurations.maybeCreate("compileShaded")
+configurations.getByName("compileOnly").extendsFrom(shade)
 dependencies {
     implementation("org.jetbrains.kotlin:kotlin-stdlib:1.3.72")
     implementation("org.jetbrains.kotlin:kotlin-gradle-plugin-api:1.3.72")
     implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.72")
-    implementation("com.android:zipflinger:3.6.3")
+
+    // We want a newer version of ZipFlinger for Zip64 support but don't want to incur that cost on
+    // consumers, so we shade it.
+    shade("com.android:zipflinger:4.1.0-alpha09") {
+        // ZipFlinger depends on com.android.tools:common which has breaking changes for AGP.
+        exclude(group = "com.android.tools")
```

Comment:
@fabiensanglard ZipFlinger appears to depend on both guava and com.android.tools:common without using anything from either. Worth filing a bug report to remove the dependencies?